cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -3,247 +3,378 @@
+                position: absolute;
+                left: 50%;
+                width: 0;
+                height: @margin__em;
+                border-left: @spine-width__px solid @spine-color;
+            }
+        });
     }
+}
+
+/* Lines of the org chart are drawn with before & after
+   pseudo elements applied to .org-chart_branches & .org-chart_branch.
+*/
+.org-chart_branches {
+    padding-top: @grid_gutter-width;
 
-    #content-slider-container {
+    /* :before pseudo element's left border draws a centered vertical line
+       down from .org-chart_root to .org-chart_branches.
+    */
+    .respond-to-min( @bp-sm-min, {
         position: relative;
-        margin-left: -10px;
 
-        #content-slider {
-            display: none;
+        &:before{
+            // Whitespace in content so element can have dimensions set.
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 50%;
+            width: 0;
+            height: @margin__em;
+            border-left: @spine-width__px solid @spine-color;
         }
+    } );
+}
 
-        .slick-slide {
-            margin-left: 10px;
-        }
+.org-chart_categories {
+    .link_item {
+        margin-bottom: 0;
+        background: @gray-5;
+        border-top: 1px solid @node-border-color;
 
-        .slick-slider {
-            margin-bottom: 0;
+        &:last-child {
+            border-bottom: 1px solid @node-border-color;
         }
     }
 
-    .content-hide {
-        display: none;
-        background-color: inherit;
-        margin-bottom: 20px;
+    .link_text {
+       border: none;
     }
+}
 
-    .content-show,
-    .content-hide {
+.org-chart_node {
+    .webfont-regular();
+    position:relative;
+    box-sizing: border-box;
+    padding: @margin__em @margin__em / 2;
+    margin: 0 auto @margin__em / 2;
+    border-bottom: @node-border-width__px solid @node-border-color;
+    background: @white;
+
+    // nested .org-chart_node blocks have different borders.
+    .org-chart_nodes {
+        border: none;
+
+        .org-chart_node {
+            padding-top: @margin__em / 2;
+            border-width: 1px;
+            margin-bottom: 0;
 
-        .respond-to-max(@bp-xs-max, {
-            .cf-icon,
-            &:before,
             &:after {
-                color: @pacific;
+                border: none;
             }
 
-            &:focus {
-                outline: thin dotted;
+            &:last-child {
+                border: none;
             }
 
-            &:hover {
-                border-color: @pacific-60;
-                z-index: 2;
+            .respond-to-max(@bp-xs-max, {
+                margin: 0;
+            });
+        }
+    }
 
-                .cf-icon,
-                &:before,
-                &:after {
-                    color: @pacific-60;
-                }
-            }
+    &:not(:last-child):after {
+        // Whitespace in content so element can have dimensions set.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/dca185a0d8c0205efa146b0695e88fa92ea3df97.